test/src/page_models/school/schoolDetailsPage.js

```diff
@@ -159,15 +159,15 @@ class SchoolDetailsPage {
     await t.typeText(this.physicalAddressCity, physicalAddress.city);
     log.info('Physical city updated');
 
-    await t.click(this.physicalAddressProvince);
-    await t.selectText(this.physicalAddressProvince).pressKey('delete');
-    await t.typeText(this.physicalAddressProvince, physicalAddress.provinceDescription);
-    log.info('Physical province updated');
+    await t
+    .click(this.physicalAddressProvince)
+    .click(Selector('div[role="listbox"]').filterVisible().find('div').withText(physicalAddress.provinceDescription));
+    log.info('Mailing province updated');
 
-    await t.click(this.physicalAddressCountry);
-    await t.selectText(this.physicalAddressCountry).pressKey('delete');
-    await t.typeText(this.physicalAddressCountry, physicalAddress.countryDescription);
-    log.info('Physical country updated');
+    await t
+    .click(this.physicalAddressCountry)
+    .click(Selector('div[role="listbox"]').filterVisible().find('div').withText(physicalAddress.countryDescription));
```

Comment:
While continuing a chain on a new line, I like to indent over 1 level for clarity.  Thoughts?